mypyc/test/test_run.py

```diff
@@ -54,15 +54,11 @@
     'run-dunders.test',
     'run-singledispatch.test',
     'run-attrs-non-auto.test',
+    'run-attrs.test',
```

Comment:
Maybe we could just use one test file for these cases?

Reply:
done